.gitlab/kitchen_deploy.yml

```diff
@@ -36,10 +36,25 @@
   - filename=$(ls datadog-signing-keys*.deb); mv $filename datadog-signing-keys_${DD_PIPELINE_ID}.deb
   - popd
 
+# Avoid simultaneous writes on the repo metadata file that made kitchen tests fail before
+.deploy_deb_resource_group-a6: &deploy_deb_resource_group-a6
+  resource_group: deploy_deb_a6
+
+.deploy_deb_resource_group-a7: &deploy_deb_resource_group-a7
+  resource_group: deploy_deb_a7
+
+.deploy_rpm_resource_group-a6: &deploy_rpm_resource_group-a6
+  resource_group: deploy_rpm_a6
+
+.deploy_rpm_resource_group-a7: &deploy_rpm_resource_group-a7
+  resource_group: deploy_rpm_a7
```

Comment:
I believe the rpm parts aren't needed.

The reason why the deb parts are needed are due to the deb repository structure:
```
apttesting.datad0g.com/
  dists/
    pipeline-<pipeline id>-<agent version>/
      Release <- Shared metadata, written to by both amd64 and arm64 deploy jobs
      Release.gpg
      <agent version>/
        binary-amd64/
          Packages <- Specific metadata, written to by the amd64 deploy job
        binary-arm64/
          Packages <- Specific metadata, written to by the arm64 deploy job
```

In the rpm repo, we don't have such shared metadata:
```
yumtesting.datad0g.com/
  testing/
    pipeline-<pipeline id>-<agent version>/
      <agent version>/
        aarch64/
          repodata/ <- Specific metadata, written to by the amd64 deploy job
        amd64/
          repodata/ <- Specific metadata, written to by the arm64 deploy job
```

Reply:
Thanks for the detailed explanation ! Fixed in [0126fa2](https://github.com/DataDog/datadog-agent/pull/14398/commits/0126fa27e28b95056d4925a838230c3a634e853a)